src/pages/AuthPage/AuthPage.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { useStore, useDispatch } from "react-redux";
+import axios from 'axios';
+
+import { userSignUp, userLogin } from "../../ducks/authIntoSystem";
+
+import AuthForm from "../../components/AuthForm";
+
+import Img1 from '../../assets/auth-three-people-image.svg';
+import Img2 from '../../assets/auth-games-image.svg';
+import Img3 from '../../assets/auth-robot-image.svg';
+import Img4 from '../../assets/auth-mother-daughter-image.svg';
```

Comment:
As these images don't rely to vector graphics I think it's not correct to save them as svg. I suggest `webp` or `avif` format. If you want to cover old browsers you could use `picture` tag and provide different img formats.

PS. Img1 isn't a well-described name)